stories/feedback/ProgressBar.stories.js

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { withInfo } from '@storybook/addon-info';
-import { action } from '@storybook/addon-actions';
 import StoryContainer from '../storyHelpers/StoryContainer';
 
 import { ProgressBar } from '@sparkpost/matchbox';
 
 storiesOf('Feedback|ProgressBar', module)
-  .addDecorator((getStory) => (
-    <StoryContainer>{ getStory() }</StoryContainer>
-  ))
-  .add('Default', withInfo()(() => (
-    <ProgressBar completed={54} color='red' />
-  )));
+  .addDecorator(getStory => <StoryContainer>{getStory()}</StoryContainer>)
+  .add(
+    'Default',
+    withInfo()(() => (
+      <div>
+        <ProgressBar completed={54} color="purple" />
```

Comment:
the `color` prop is deprecated, no? Might be confusing leaving it in from a documentation point of view.